kinder/ci/workflows/ha-cert-copy-with-external-etcd.yaml

```diff
@@ -0,0 +1,119 @@
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used for creating a HA
+  cluster with an external etcd cluster.
+vars:
+  kubernetesVersion: v1.14.1
+  controlPlaneNodes: 2
```

Comment:
we just changed the other workflows to have 3 cp nodes.
possibly can be harcoded as `--control-plane-nodes=3` too.